tests/lib/config/config-validator.ts

```diff
@@ -28,15 +28,15 @@ test('if config has an invalid schema, it should return false', (t) => {
     t.false(valid);
 });
 
-test(`if there is a rule in the config that doesn't exist, it should return false`, (t) => {
-    const config = _.cloneDeep(validConfig);
+// test(`if there is a rule in the config that doesn't exist, it should return false`, (t) => {
```

Comment:
I just moved this test to the `sonar.ts` ones because that's where we do the rule validation.